### DISCUSSION TITLE
Welcome to pulp Discussions!

### DISCUSSION THREAD

**Discussion by pchtsp** [coin-or/pulp#338] [General]
## Welcome!

We've decided to move the support for / conversations about pulp from the google group to github Discussions, following other coin or projects like [cbc](https://github.com/coin-or/Cbc/discussions/331).

The idea is to use Discussions as a place to connect with other members of our community in order to:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.

The idea is to keep using github Issues for real problems of the library and all the rest handle it through Discussions. We hope this will make communication more organized by having everything (most things) in the same place.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rafalQA** [comment]
Hi all !!!
I created some Java application which makes rest request to Flask Python service which uses Pulp to solve some problem with default CBC. Localy on my windows its works perfectly. But currently I'm moving all componets on Docker . And when Pulp is trying to invoke cbc solver I getting this issue: 

![image](https://user-images.githubusercontent.com/14124520/94558708-81a32200-0260-11eb-8992-b616ed61f2b6.png)

I'm supposing its takes wrong path, because in the folder "apis" (you can notice on the scree above), there is no folder "solvedir" :

![image](https://user-images.githubusercontent.com/14124520/94559248-3c332480-0261-11eb-9efc-25540b98062f.png)


The cbc for 64 is in :

![image](https://user-images.githubusercontent.com/14124520/94559507-a4820600-0261-11eb-9b66-8a4b15fb70f3.png)


So I think it could be a reason. In pulp.cfg.linux file it looks the path is correct. But still get this issue. I also try to chmod 777, without result. 

I'm not able aslo run cbc directly on linux (on Windows I can run without any problem ):

![image](https://user-images.githubusercontent.com/14124520/94560795-684fa500-0263-11eb-8cb7-02c26265cdeb.png)


![image](https://user-images.githubusercontent.com/14124520/94560696-4a824000-0263-11eb-8f01-f7a00c739d5d.png)


My docker file is:

![image](https://user-images.githubusercontent.com/14124520/94559835-18bca980-0262-11eb-97f3-6d0a6644b9b2.png)

I would be very grateful if someone could give me some advice :)

**Reply by pchtsp** [reply]
Maybe check this project where I'm working: https://github.com/baobabsoluciones/corn
It's a server-side solver api built on top of airflow, pulp and flask that is already configured for docker deployment. 

Feel free to use it, fork it and contribute to it.

**Reply by rafalQA** [reply]
Hi, I just chenged base image from python:3.7-alpine  to python:3.7, and now its works, But I'm very interesting yours project, I'm going to look :)
